src/views/home/index.js

```diff
@@ -3,37 +3,54 @@ import React, { useEffect, useState, useContext } from 'react'
 import { AuthContext } from '../../providers/use-auth'
 import { Navigate } from "react-router-dom";
 import './home.css'
-import { getDog } from '../../utils/fetch-request';
+import { useInitDog } from '../../hooks';
+import { usePreloadDog } from '../../hooks';
 
 export const Home = () => {
   const { user } = useContext(AuthContext)
   const [left, setLeft] = useState(null)
   const [mid, setMid] = useState(null)
   const [right, setRight] = useState(null)
+  const dogs = useInitDog()
+  const [preloadDogs, setPreloadDogs] = usePreloadDog()
+  const preLoadImg = new Image()
```

Comment:
Btw @wood-crafter you don't need this variable here...!
You can just `new Image()` in the `useEffect` below, right...?